### PR TITLE
doc: update links to the documentation

### PR DESCRIPTION
**Pull request by jbech-linaro**
Fix outdated links.

Fixes: #414

Signed-off-by: Joakim Bech <joakim.bech@linaro.org>

### CODE REVIEW

**Comment by jbech-linaro** [comment]
Wanted to add the Suggested-by to the guy in #414, but couldn't find the information and he seems to have gone missing lately.

File: common.mk [hunk 1]
```diff
@@ -208,6 +208,7 @@ BUILDROOT_TOOLCHAIN=toolchain-aarch$(COMPILE_NS_USER)-legacy
 endif
 endif
 
+BR2_PACKAGE_INOTIFY_TOOLS ?= y
```
Comment (jforissier) [inline common.mk:211]: Unrelated?

Comment (jbech-linaro) [reply common.mk:211]: Indeed, got added by mistake.

**Comment by jbech-linaro** [comment]
Squashed, rebased and tag(s) applied (should be) ready for merge! Thanks!
